uat/custom-components/src/main/java/com/aws/greengrass/artifacts/LogGenerator.java

```diff
@@ -59,8 +59,8 @@ private void generateLogs() throws InterruptedException {
         Logger logger = configureLogger();
 
         for (int i = 1; i <= numberOfLogs; i++) {
-            String logLine = String.format("(seq: %d, name: %s)", i, logFileName);
-            logger.info(logLine);
+            String logLine = String.format("(seq: %d)", i);
```

Comment:
This is to make sure that the log message length is less than 70 bytes (34 more bytes are added per log event) and  so LM will try to upload 10,000+ log events at once (10000*104 = ~1,040,000). Otherwise, max limit is reached (1024*1024 = ~1,048,576) for <10,000 logs and LM will upload log events  less than 10,000.